recipe/meta.yaml

```diff
@@ -1,40 +1,38 @@
 {% set version = "0.82" %}
-{% set sha256 = "3f8375f2ced161a203c3521520fdeb8b8b5b088987c98b609abdff7408b78a4c" %}
 
 package:
   name: pybufr-ecmwf
   version: {{ version }}
 
 source:
-  fn: pybufr-ecmwf-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/p/pybufr-ecmwf/pybufr-ecmwf-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 3f8375f2ced161a203c3521520fdeb8b8b5b088987c98b609abdff7408b78a4c
 
 build:
-  number: 1
+  number: 1001
+  skip: True  # [win]
   script:
     - export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib  # [osx]
     - export LDFLAGS="-undefined dynamic_lookup -bundle"  # [osx]
-    - python setup.py install
-  skip: True  # [win]
+    
+    --fortran-compiler and --c-compiler
```

Comment:
Should this line be here or is it missing something?

Reply:
if you use these options to the setup.py build or install stage, they should be followed by the compiler executable including the full path to its location. This is not standard python, but a customisation in this setup script.
See also: "python setup.py build --help"
But I don't really understand why you would like to support different compilers in the conda context. pybufr-ecmwf should be compiled using the exact same gcc(+gfortran) version that was used to compile the python interpreter otherwise it will not be able to load the shared object file as python module.